package.json

```diff
@@ -55,7 +56,8 @@
     "hoist-non-react-statics": "^1.2.0",
     "immutable": "^3.8.1",
     "jwt-decode": "^2.2.0",
-    "lodash": "^4.17.4"
+    "lodash": "^4.17.4",
+    "opencollective": "^1.0.3"
```

Comment:
this is optional so if it's an issue for you feel free to remove it.
you can also take a more native approach and do something like Preact did:
https://github.com/developit/preact/blob/92f65dfddfe038c4584f5f35e8de93a66c98dda4/package.json#L32